hera_mc/tests/test_rtp.py

```diff
@@ -2,462 +2,481 @@
-        exp_columns['stop_time'] = int(floor(exp_columns['stop_time'].gps))
-        expected = RTPTaskResourceRecord(**exp_columns)
-
-        result = self.test_session.get_rtp_task_resource_record(
-            starttime=self.task_resource_columns['start_time'] - TimeDelta(2, format='sec'),
-            obsid=self.record_columns['obsid'])
-        self.assertEqual(len(result), 1)
-        result = result[0]
-        self.assertTrue(result.isclose(expected))
-
-    def test_errors_rtp_task_resource_record(self):
-        self.test_session.add_obs(*self.observation_values)
-        obs_result = self.test_session.get_obs()
-        self.assertTrue(len(obs_result), 1)
-
-        fake_vals = [1, 'a', 2, 'b', 3, 'c']
-        self.assertRaises(ValueError, self.test_session.add_rtp_task_resource_record, *fake_vals)
-        # test case where start_time is an astropy.time object, but stop_time isn't
-        fake_vals2 = [1, 'a', Time.now(), 'b', 3, 'c']
-        self.assertRaises(ValueError, self.test_session.add_rtp_task_resource_record, *fake_vals2)
-
-        self.test_session.add_rtp_task_resource_record(*self.task_resource_values)
-
-        self.assertRaises(ValueError, self.test_session.get_rtp_task_resource_record,
-                          task_name=self.task_resource_columns['task_name'], most_recent=False)
-
-        self.assertRaises(ValueError, self.test_session.get_rtp_task_resource_record,
-                          most_recent=False)
-
-
-if __name__ == '__main__':
-    unittest.main()
+time = Time.now() - TimeDelta(30 * 60, format='sec')
+obsid = utils.calculate_obsid(time)
+observation_names = ['starttime', 'stoptime', 'obsid']
+observation_values = [time, time + TimeDelta(10 * 60, format='sec'), obsid]
+observation_columns = dict(zip(observation_names, observation_values))
+status_names = ['time', 'status', 'event_min_elapsed',
+                'num_processes', 'restart_hours_elapsed']
+status_values = [time, 'happy', 3.6, 8, 10.2]
+status_columns = dict(zip(status_names, status_values))
+
+event_names = ['time', 'obsid', 'event']
+event_values = [time, obsid, 'queued']
+event_columns = dict(zip(event_names, event_values))
+
+record_names = ['time', 'obsid', 'pipeline_list', 'rtp_git_version',
+                'rtp_git_hash', 'hera_qm_git_version', 'hera_qm_git_hash',
+                'hera_cal_git_version', 'hera_cal_git_hash',
+                'pyuvdata_git_version', 'pyuvdata_git_hash']
+record_values = [time, obsid, 'sample_pipe', 'v0.0.1', 'lskdjf24l', 'v0.1.0',
+                 'abcd34d', 'v1.0.0', 'jkfldi39', 'v2.0.0', 'fjklj828']
+record_columns = dict(zip(record_names, record_values))
+
+task_resource_names = ['obsid', 'task_name', 'start_time', 'stop_time',
+                       'max_memory', 'avg_cpu_load']
+task_resource_values = [obsid, 'OMNICAL', time,
+                        time + TimeDelta(10 * 60, format='sec'), 16.2, 1.01]
+task_resource_columns = dict(zip(task_resource_names, task_resource_values))
```

Comment:
Another instance where a module scoped fixture may be appropriate

Reply:
fixed.